src/dnx/dnx.cpp

```diff
@@ -176,7 +177,7 @@ void ExpandArgument(const dnx::char_t* value, std::vector<const dnx::char_t*>& e
 
 bool ExpandCommandLineArguments(size_t nArgc, dnx::char_t** ppszArgv, size_t& nExpandedArgc, dnx::char_t**& ppszExpandedArgv)
 {
-    size_t pivot_parameter_idx = FindAppBaseOrNonHostOption(nArgc, ppszArgv);
+    size_t pivot_parameter_idx = FindOption(nArgc, ppszArgv, _X("--appbase"));
```

Comment:
I didn't want to change this because it was already there but I had the same thought. I can just try it and see if it works :P